tasks/go_deps.py

```diff
@@ -115,3 +115,110 @@ def send_count_metrics(
         print(color_message("Sending metrics to Datadog", "blue"))
         send_metrics(series=series)
         print(color_message("Done", "green"))
+
+
+BINARY_TO_TEST = ["serverless"]
+
+
+@task
+def test_dependencies_list(
```

Comment:
Same as for the other task, the namespace is `go-deps` so it could just be `test_list` to avoid redundancy